client/src/pages/ProfileDetails/ReviewForm/ReviewForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect, useContext } from 'react';
+import { Box, Typography, Card, CardMedia, CardContent, Button, TextField } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+import useStyles from './useStyles';
+import { createReview } from '../../../helpers/APICalls/createReview';
+import { useParams } from 'react-router-dom';
+import { AuthContext } from '../../../context/useAuthContext';
+import { SnackBarContext } from '../../../context/useSnackbarContext';
+
+const ReviewForm = (): JSX.Element => {
+  const classes = useStyles();
+  const [rating, setRating] = useState<number | null>(0);
+  const [description, setDescription] = useState('');
+  const { id }: { id: string } = useParams();
+  const { loggedInUser } = useContext(AuthContext);
+  const { updateSnackBarMessage } = useContext(SnackBarContext);
+
+  const submitReview = () => {
+    if (loggedInUser) {
+      createReview(id, loggedInUser.id, rating, description).then((data) => {
+        if (data.success) {
+          updateSnackBarMessage('Review submitted successfully');
+          window.location.reload();
```

Comment:
As opposed to reloading the entire window we can return the new review as a response and append it to our existing state